.github/release.yml

```diff
@@ -0,0 +1,27 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Install Node.js
+      uses: actions/setup-node@v1
+      with:
+        node-version: 14.x
+    - run: npm install
+    - name: Publish package 
+      run: npx vsce publish patch
```

Comment:
Q: Wenn ich das richtig verstehe, wird so zuerst die Versionsnummer aktualisiert und erst dann die Extension erstellt. Wäre es nicht klarer, die Version auf dem aktuellen Repo-Stand zu builden, und im Anschluss per `npm version patch` die Version zu aktualisieren (und zu pushen)?

Reply:
Würde sicherlich Sinn machen. Kannte die Option gar nicht. Ich werde das noch entsprechend anpassen.

Reply:
`npm version patch` erstellt auf der neu erreichten Version einen Tag. Daher sehe ich es nicht mehr als sinnvoll an.

Reply:
`npm --no-git-tag-version version patch` FTW?

— https://docs.npmjs.com/cli/v7/commands/npm-version

Reply:
Die standard commit message würde dann weiterhin die neu erreichte version tragen. Ist das dann nicht verwirrend?

Reply:
Das kommt drauf an in welcher Reihenfolge die Sache geschehen. Wenn der Ablauf ist

1. Release für die im Repo gekennzeichnete Version erstellen und Git-Tag erstellen
2. ggf. GitHub-Release erstellen
3. Version als Vorbereitung für den nächsten Release patchen

, dann passt die Beschreibung.

Reply:
Ich denke jetzt sollte es passten. Jedoch frage ich mich bei der Verwendung all dieser flags auf `npm version` ob wir es hier gemäss dem gängigen Verständnis verwenden.